### DISCUSSION TITLE
请问大佬们有没有支持d3d11 gpu super resolution的解决方案

### DISCUSSION THREAD

**Discussion by SPiCa-P** [hooke007/mpv_PlayKit#469] [Q&A 问答]
现在potplayer支持d3d11 gpu super resolution，我在mpv似乎没有找到使用方法。

在这个讨论串中：
https://github.com/mpv-player/mpv/issues/11390
我注意到有人已经尝试添加支持，但是实际使用时mpv会闪退，并且没有error logs

**Comment by hooke007** [comment]
不要重复提问已有的问题。
问之前可以先搜一下

如果只在pr里有的东西，那你应该去pr而不是这里反馈
